roles/openshift_node/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-openshift_kubeconfig_path: '~/.kube/config'
+openshift_kubeconfig_path: "{{ openshift_kubeconfig_path | default('~/.kube/config') | realpath }}"
```

Comment:
Why would we need `default(` here? These are role defaults, so user should be able to redefine those, aren't they?

Reply:
If the user doesn't provide anything, default to ~/.kube/config and expand to full path.
If the user does provide something, still pass it through realpath to expand to a full path.

Reply:
The problem is that `oc` doesn't deal with `~` so the role default doesn't work.

Reply:
Actually, I'm worried that this may cause a recursive loop. Looking to see if we have any other examples of this pattern.

Reply:
Why not using `realpath` in the task where `openshift_kubeconfig_path` is being used?

Reply:
Yep, busted. ...WIP